portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase

```diff
@@ -33,6 +33,44 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test StructuredContentDraftContainsModifiedPriorityValue {
+		property portal.acceptance = "true";
+
+		task ("Given in Headless Admin Content I create a structuredContent draft with with a priority field set") {
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
+
+			var responseToParse = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "Text",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				priority = "1",
+				title = "Title");
+		}
+
+		task ("When edit a web content with different priorities") {
```

Comment:
Maybe we can change it to `When a structured content draft priority is modified` or something similar with priorities in singular? :)